### ISSUE TITLE
Checkpoints of finetuned AVSR models without VoxCeleb2 data?

### ISSUE THREAD

**Issue by ALIVE321** [facebookresearch/av_hubert#64]
Hi, thanks for your awesome work!
I'm wondering if you have any plan to release the checkpoints of `base/large_noise_lrs3_pt_noise_433h/30h_ft`, i.e. pretrained and finetuned only on LRS3? 
It will be useful since we're planning to take your work as a strong model for comparison in the AVSR task on LRS3.

**Comment by chevalierNoir** [comment]
Hi,

Currently, all our AVSR models are pretrained on VoxCeleb2+LRS3. We might pretrain/fine-tune a base model for AVSR using LRS3 only in the future but we haven't got a specific timeline for that yet. 

Btw, following our pretraining pipeline, the AVSR model is pretrained for one iteration only. If you want to pretrain an AVSR model with LRS3 only, you can use the publicly available checkpoint of the second-to-last iter to produce clusters and do one round of pretraining with those clusters.

**Comment by ALIVE321** [comment]
> Hi,
> 
> Currently, all our AVSR models are pretrained on VoxCeleb2+LRS3. We might pretrain/fine-tune a base model for AVSR using LRS3 only in the future but we haven't got a specific timeline for that yet.
> 
> Btw, following our pretraining pipeline, the AVSR model is pretrained for one iteration only. If you want to pretrain an AVSR model with LRS3 only, you can use the publicly available checkpoint of the second-to-last iter to produce clusters and do one round of pretraining with those clusters.

Hi, 
Seems I misunderstood something in Table 4 of the first paper "[Learning Audio-Visual Speech Representation by Masked Multimodal Cluster Prediction](https://arxiv.org/abs/2201.02184)".
So the first line `unlabeled=30 labeled=433 wer=4.9` in the last block of Table 4 (`ASR WER A/MFCC->AV Base`) means that, the audio-hubert is initially pretrained on Vox+LRS3(audio only), and gets further pretrained on LRS3_433h(audio-visual clusters from av-hubert) and finetuned on LRS3_30h(ASR task)?

If that's the truth, I can understand them now. SORRY for my mistakes, and THANKs for your kind replies.

**Comment by chevalierNoir** [comment]
> > Hi,
> > Currently, all our AVSR models are pretrained on VoxCeleb2+LRS3. We might pretrain/fine-tune a base model for AVSR using LRS3 only in the future but we haven't got a specific timeline for that yet.
> > Btw, following our pretraining pipeline, the AVSR model is pretrained for one iteration only. If you want to pretrain an AVSR model with LRS3 only, you can use the publicly available checkpoint of the second-to-last iter to produce clusters and do one round of pretraining with those clusters.
> 
> Hi, Seems I misunderstood something in Table 4 of the first paper "[Learning Audio-Visual Speech Representation by Masked Multimodal Cluster Prediction](https://arxiv.org/abs/2201.02184)". So the first line `unlabeled=30 labeled=433 wer=4.9` in the last block of Table 4 (`ASR WER A/MFCC->AV Base`) means that, the audio-hubert is initially pretrained on Vox+LRS3(audio only), and gets further pretrained on LRS3_433h(audio-visual clusters from av-hubert) and finetuned on LRS3_30h(ASR task)?
> 
> If that's the truth, I can understand them now. SORRY for my mistakes, and THANKs for your kind replies.

Hi,

The model that you refer to (`labled=30h, unlabeled-433h, wer=4.9`) is an audio-HuBERT pretrained with 433h of audio (LRS3 only) and fine-tuned with 30h of audio. The clusters used for pretraining are from an AV-HuBERT (pretrained on LRS3 only). For those two models, the AV-HuBERT is publicly available but the audio-HuBERT is not.

On the other hand, our AVSR models (i.e., audio-visual speech recognition models in the second paper [Robust Self-Supervised Audio-Visual Speech Recognition](https://arxiv.org/abs/2201.01763)), are all pre-trained with Vox+LRS3.

**Comment by ALIVE321** [comment]
> > > Hi,
> > > Currently, all our AVSR models are pretrained on VoxCeleb2+LRS3. We might pretrain/fine-tune a base model for AVSR using LRS3 only in the future but we haven't got a specific timeline for that yet.
> > > Btw, following our pretraining pipeline, the AVSR model is pretrained for one iteration only. If you want to pretrain an AVSR model with LRS3 only, you can use the publicly available checkpoint of the second-to-last iter to produce clusters and do one round of pretraining with those clusters.
> > 
> > 
> > Hi, Seems I misunderstood something in Table 4 of the first paper "[Learning Audio-Visual Speech Representation by Masked Multimodal Cluster Prediction](https://arxiv.org/abs/2201.02184)". So the first line `unlabeled=30 labeled=433 wer=4.9` in the last block of Table 4 (`ASR WER A/MFCC->AV Base`) means that, the audio-hubert is initially pretrained on Vox+LRS3(audio only), and gets further pretrained on LRS3_433h(audio-visual clusters from av-hubert) and finetuned on LRS3_30h(ASR task)?
> > If that's the truth, I can understand them now. SORRY for my mistakes, and THANKs for your kind replies.
> 
> Hi,
> 
> The model that you refer to (`labled=30h, unlabeled-433h, wer=4.9`) is an audio-HuBERT pretrained with 433h of audio (LRS3 only) and fine-tuned with 30h of audio. The clusters used for pretraining are from an AV-HuBERT (pretrained on LRS3 only). For those two models, the AV-HuBERT is publicly available but the audio-HuBERT is not.
> 
> On the other hand, our AVSR models (i.e., audio-visual speech recognition models in the second paper [Robust Self-Supervised Audio-Visual Speech Recognition](https://arxiv.org/abs/2201.01763)), are all pre-trained with Vox+LRS3.

OK thanks, that makes sense too. Looking forward to your future works.
